src/keyboard.rs

```diff
@@ -323,6 +323,15 @@ pub enum KeyCode {
   KeyZ,
   /// <kbd>-</kbd> on a US keyboard.
   Minus,
+  /// <kbd>Shift</kbd>+<kbd>=</kbd> on a US keyboard.
+  ///
+  /// When used as a menu accelerator this is displayed as <kbd>+</kbd>, and on macOS
+  /// and Windows the accelerator can be used without pressing the <kbd>Shift</kbd> key.
+  /// On Linux the <kbd>Shift</kbd> key is still required with a US keyboard layout.
+  ///
+  /// `Plus` does not work as a value for keyboard accelerators outside menus, on
+  /// US keyboards or others where <kbd>+</kbd> requires a modifier key.
```

Comment:
```suggestion
  ///
  /// `Plus` does not work as a value for keyboard accelerators outside menus where <kbd>+</kbd> requires a modifier key.
```

Reply:
I'm not sure that quite captures the intent of what I meant to say – how about this?

```suggestion
  /// `Plus` does not work as a value for keyboard accelerators outside menus on
  /// keyboards where <kbd>+</kbd> requires a modifier key.
```

Reply:
sounds good